bleak/backends/corebluetooth/PeripheralDelegate.py

```diff
@@ -87,14 +87,21 @@ def futures(self) -> Iterable[asyncio.Future]:
     @objc.python_method
     async def discover_services(self, use_cached: bool = True) -> NSArray:
         if self._services_discovered_future.done() and use_cached:
-            return self.peripheral.services()
+            services = self.peripheral.services()
+            if services is None:
+                logger.debug("Services lost retrying")
```

Comment:
We should be able to do without this debug log (especially if the previous suggestion is taken). In the case that we have to call `discoverServices_()`, the delegate callback should trigger a log anyway, so we have another way to know that code path is taken.